Nodejs/Product/TestAdapter/SerializationHelpers.cs

```diff
@@ -40,19 +40,22 @@ public TestCaseObject()
         this.testFile = string.Empty;
         this.workingFolder = string.Empty;
         this.projectFolder = string.Empty;
+        this.configPath = string.Empty;
     }
 
-    public TestCaseObject(string framework, string fullyQualifiedName, string testFile, string workingFolder, string projectFolder)
+    public TestCaseObject(string framework, string fullyQualifiedName, string testFile, string workingFolder, string projectFolder, string configPath)
```

Comment:
It seems strange to have this ctor when the properties are writeable.  Anyone consuming it could just use an object initializer, no?  And that would eliminate the need to keep the signature up-to-date?

Reply:
I suppose in a way the consumer will be force to consider the variable value when initializing, but I don't have a particular preference. I could change it if you feel is more readable.